stratum/hal/lib/barefoot/bfrt_table_manager.cc

```diff
@@ -51,7 +53,8 @@ ::util::Status BfrtTableManager::PushForwardingPipelineConfig(
     ASSIGN_OR_RETURN(
         P4Annotation annotation,
         p4_info_manager->GetSwitchStackAnnotations(reg.preamble().name()));
-    if (annotation.register_reset_interval_ms()) {
+    if (annotation.register_reset_interval_ms() &&
+        mode_ == OPERATION_MODE_STANDALONE) {
```

Comment:
It would be helpful to print a warn log saying that register reset is disabled because of simulation mode